Library/Formula/re2.rb

```diff
@@ -16,11 +16,15 @@ class Re2 < Formula
     sha256 "941571b08e34921134c2fc15b5e855f7c2da5a882fd262766fa01dac988990f3" => :mavericks
   end
 
+  needs :cxx11 unless OS.mac?
```

Comment:
Yeah, C++ 11 is used by the formula by default on OS X, presumably due to XCode.